packages/react-forms/src/InputDate/__mocks__/InputDateWrapper.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+import React, { useState, useCallback, Fragment } from 'react';
+import InputDate from '../index';
+
+type InputDateWrapperProps = {
+  defaultValue: string,
+  defaultCalendarValue?: Date,
+  onChange?: string => void,
+  onCalendarChange?: Date => void,
+  isOpen?: boolean,
+};
+
+const InputDateWrapper = ({
+  onChange,
+  defaultValue,
+  onCalendarChange,
+  defaultCalendarValue,
+  isOpen: isOpenProp,
+  ...props
+}: InputDateWrapperProps) => {
+  const [isOpen, setOpen] = useState(isOpenProp);
+  const [value, setValue] = useState(defaultValue);
+  const [calendarValue, setCalendarValue] = useState(
+    onCalendarChange ? defaultCalendarValue : new Date()
+  );
+
+  const onChangeHandler = useCallback(
+    value => {
+      onChange && onChange(value);
+      setValue(value);
+    },
+    [setValue, onChange]
+  );
+
+  const handleCalendarChange = useCallback(
+    value => {
+      onCalendarChange && onCalendarChange(value);
+      setCalendarValue(value);
+    },
+    [setCalendarValue, onCalendarChange]
+  );
+
+  const handleClose = useCallback(() => {
+    setOpen(false);
```

Comment:
no need for the extra curly braces, I'd suggest to try to write the code without them and add them only when they are needed, it should help you to keep the code cleaner. 🧹

Reply:
I forgot to remove it. 😭